Add login/sigup pages and routes

MinaKhamesi opened the pull request:
<img width="1440" alt="Screen Sho
<img width="1440" alt="Screen Shot 2021-03-09 at 1 23 03 PM" src="https://user-images.githubusercontent.com/53087960/110519360-362c5400-80db-11eb-8966-34efb411b00c.png">
<img width="745" alt="Screen Shot 2021-03-09 at 1 23 45 PM" src="https://user-images.githubusercontent.com/53087960/110519374-3debf880-80db-11eb-855d-b5dbea07113a.png">
<img width="745" alt="Screen Shot 2021-03-09 at 1 24 58 PM" src="https://user-images.githubusercontent.com/53087960/110519394-42181600-80db-11eb-832f-f27e6c7e7405.png">
<img width="745" alt="Screen Shot 2021-03-09 at 1 25 08 PM" src="https://user-images.githubusercontent.com/53087960/110519420-4ba17e00-80db-11eb-949a-6c058a0a8491.png">



t 2021-03-09 at 1 22 34 PM" src="https://user-images.githubusercontent.com/53087960/110519275-1bf27600-80db-11eb-93b8-9587e41978b7.png">
forms and banners added, form validation added using formik and yup libraries, font and logo added to public folder and theme edited to use the proper font family and secondary color changed to be light orange provided in specs, landing route will redirect to signup page and login and signup routes added to App.js @jessdesr @dproc96

MinaKhamesi committed:
Add login/sigup pages and routes

forms and banners added, form validation added using formik and yup libraries, font and logo added to public folder and theme edited to use the proper font family and secondary color changed to be light orange provided in specs, landing route will redirect to signup pageand login and signup routes added to App.js

MinaKhamesi committed:
Delete logo.png on public/images

logo.png is going to be logo.jpg in next commits so  deleted for now to prevent a conflict. SignUp and login pages modified accordingly. A typo in signUp was fixed too

dproc96 commented on client/src/AuthRoutes/components/Formik/styles/Input.style.js:
```diff
@@ -0,0 +1,17 @@
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+	formGroup: {
+		marginBottom: 20,
+	},
+	label: {
+		fontWeight: 900,
+		fontSize: 12,
+	},
+	input: {
+		margin: '3px 0',
+		borderRadius: '0',
+	},
+}));
```
You could move these into the component file or leave them separate but you should consult with the team about preferred convention

rudream replied:
I agree with Dan, I think it would be better to have the useStyles in the same file as the component. That's how I usually set it up. I find that it makes it easier to make small adjustments without having to switch back and forth between files.

MinaKhamesi replied:
Thanks Dan. 
I will add styles to components and get rid of the extra files and folders.
About prettier, we discussed on standup meetings, one of teammates is going to make a pull request with prettier configuration, and I guess the problem will be solved after his merge.
Other than styles being separate , the other structures like having pages and components folders inside each section, I discussed  it with Yassine, I guess this works for him too, I will check again with the team to make sure everyone is comfortable with the structure os src folder.
 I will make a commit with these changes this afternoon after our standup meeting.

dproc96 reviewed (commented):
The UI looks great! Nice work! Just a couple of thoughts about this.

At the moment your directory structure is extremely modularized. You may want to check with your team about whether to keep it as is or store it more like "/src/components" and "/src/pages"

Additionally, check with your team about prettier.io settings because it looks like your tab width is 4 right now.

MinaKhamesi committed:
Fix changing size of input fielda and overflow of forms

there is no scroll on signup page anymore and the validation texts won't change the size of form anymore. Also the directory structure modified according to what we discussed in standup meeting.

MinaKhamesi committed:
Delete Auth.style.js and Insert the style into Login.js

jessdesr reviewed (approved):
Looking good!  Pull the prettier config,  update your files, and I think this can be merged now.  You can put up another pull request with the formatting changes I suggested for the helper text under the inputs

MinaKhamesi committed:
Merge branch 'dev' into feature-login-signup-pages
Prettier Config Added

MinaKhamesi committed:
Incorporate prettier settings
